db/migrate/20210312140240_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Here consider using def up and def down for migration

Reply:
Thanks, I will

Reply:
I did it